lib/send-request.js

```diff
@@ -24,19 +27,39 @@ module.exports = function(location, options, callback) {
     callback(null, request)
   }
 
+  // Prepare the request payload for forms, if available
+  if (requestParams && requestParams.length) {
+    switch (requestMethod) {
+      case "GET":
+        // Reset query string to avoid an issue with repeat submissions where checkboxes that were
+        // previously checked are incorrectly preserved
+        location = location.split("?")[0]
+
+        // Build new query string
+        requestParams.forEach(function(param) {
```

Comment:
@robinnorth Is there any reason not to build the querystring for both GET and POST in 1 step, and then depending on the method, append it to `location` or the body?

Reply:
There was indeed a reason to build them separately to begin with, as I was updating the query string parameters for GET submissions, however I've had to abandon that and discard any previously-set ones in order to properly reflect the behaviour of unchecked checkbox inputs on form submission -- they should be omitted entirely from the query string, even if they had been checked previously.

So, I will refactor this as suggested.